docs/user/01-10-architecture.md

```diff
@@ -1,22 +1,14 @@
-# Kyma Environment Broker Architecture
+# Kyma Environment Broker Target Architecture
 
-The diagram and steps describe the Kyma Environment Broker (KEB) workflow and the roles of specific components in this process:
-
-![KEB diagram](../assets/keb-arch.drawio.svg)
+![KEB architecture](../assets/target-keb-arch.drawio.svg)
 
 1. The user sends a request to create a new cluster with SAP BTP, Kyma runtime.
-
-2. KEB sends the request to create a new cluster to Runtime Provisioner.
-
-3. Runtime Provisioner creates a new cluster.
-
-4. KEB creates a GardenerCluster resource.
-
-5. Infrastructure Manager creates and maintains a Secret containing a kubeconfig.
-
-6. KEB creates a Kyma resource.
-
-7. Lifecycle Manager manages Kyma modules.
+2. KEB creates a GardenerCluster resource.
+3. Infrastructure Manager provisions a new Kubernetes cluster.
+4. Infrastructure Manager creates and maintains a Secret containing a kubeconfig.
+5. KEB creates a Kyma resource.
+6. Lifecycle Manager reads the Secret every time it's needed.
+7. Lifecycle Manager manages modules within SAP BTP, Kyma runtime.
 
 > [!NOTE]
-> In the future, Kyma Runtime Provisioner will be deprecated.  KEB will then integrate with Infrastructure Manager. To learn about the planned KEB workflow, read [Kyma Environment Broker Target Architecture](01-20-target-architecture.md).
+> Once the planned changes are implemented, the [Orchestration](../contributor/02-50-orchestration.md) document will be deprecated as irrelevant.
```

Comment:
This note doesn't make sense here, I'm afraid. It was added to the document that described the architecture planned for the future and some changes to be made to achieve the goal. Now that the changes have been implemented, the condition `Once the planned changes are implemented` has become irrelevant. If we need to warn that the Orchestration doc is to be deprecated, let's add it to the doc itself. Something along the lines: "This document will soon be deprecated. " (or something more specific).